practice-app/app/views.py

```diff
@@ -242,3 +245,60 @@ def worldTime():
                 languageCode=world_time.languageCode,
             )
         return render_template("worldtime.html")
+
```

Comment:
We decided to seperate our own codes in different files. After seperating your part into seperate file and importing them in views.py, I will merge this PR

Reply:
Ok, I have put my handler functions to a separate file named _**bored_api.py**_.